2021-10-02 11:54:52,810 INFO    MainThread:8954 [wandb_setup.py:_flush():69] setting env: {}
2021-10-02 11:54:52,810 INFO    MainThread:8954 [wandb_setup.py:_flush():69] setting login settings: {}
2021-10-02 11:54:52,810 INFO    MainThread:8954 [wandb_init.py:_log_setup():348] Logging user logs to /home/pholur/Situation_Modeling/wandb/run-20211002_115452-2t6tso8e/logs/debug.log
2021-10-02 11:54:52,810 INFO    MainThread:8954 [wandb_init.py:_log_setup():349] Logging internal logs to /home/pholur/Situation_Modeling/wandb/run-20211002_115452-2t6tso8e/logs/debug-internal.log
2021-10-02 11:54:52,810 INFO    MainThread:8954 [wandb_init.py:init():381] calling init triggers
2021-10-02 11:54:52,810 INFO    MainThread:8954 [wandb_init.py:init():388] wandb.init called with sweep_config: {}
config: {}
2021-10-02 11:54:52,810 INFO    MainThread:8954 [wandb_init.py:init():430] starting backend
2021-10-02 11:54:52,811 INFO    MainThread:8954 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-10-02 11:54:52,816 INFO    MainThread:8954 [backend.py:ensure_launched():135] starting backend process...
2021-10-02 11:54:52,926 INFO    MainThread:8954 [backend.py:ensure_launched():140] started backend process with pid: 11866
2021-10-02 11:54:52,928 INFO    MainThread:8954 [wandb_init.py:init():435] backend started and connected
2021-10-02 11:54:52,930 INFO    MainThread:8954 [wandb_init.py:init():494] updated telemetry
2021-10-02 11:54:52,931 INFO    MainThread:8954 [wandb_init.py:init():517] communicating current version
2021-10-02 11:54:53,885 INFO    MainThread:8954 [wandb_init.py:init():522] got version response upgrade_message: "wandb version 0.12.3 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-10-02 11:54:53,886 INFO    MainThread:8954 [wandb_init.py:init():530] communicating run to backend with 30 second timeout
2021-10-02 11:54:54,047 INFO    MainThread:8954 [wandb_init.py:init():557] starting run threads in backend
2021-10-02 11:54:59,054 INFO    MainThread:8954 [wandb_run.py:_console_start():1605] atexit reg
2021-10-02 11:54:59,056 INFO    MainThread:8954 [wandb_run.py:_redirect():1479] redirect: SettingsConsole.REDIRECT
2021-10-02 11:54:59,056 INFO    MainThread:8954 [wandb_run.py:_redirect():1484] Redirecting console.
2021-10-02 11:54:59,061 INFO    MainThread:8954 [wandb_run.py:_redirect():1540] Redirects installed.
2021-10-02 11:54:59,061 INFO    MainThread:8954 [wandb_init.py:init():582] run started, returning control to user process
2021-10-02 11:54:59,371 INFO    MainThread:8954 [wandb_run.py:_atexit_cleanup():1575] got exitcode: 1
2021-10-02 11:54:59,372 INFO    MainThread:8954 [wandb_run.py:_restore():1547] restore
2021-10-02 11:55:01,588 INFO    MainThread:8954 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 736
  total_bytes: 736
}

2021-10-02 11:55:02,074 INFO    MainThread:8954 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 736
  total_bytes: 736
}

2021-10-02 11:55:02,242 INFO    MainThread:8954 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 736
  total_bytes: 736
}

2021-10-02 11:55:03,097 INFO    MainThread:8954 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 736
  total_bytes: 11015
}

2021-10-02 11:55:03,201 INFO    MainThread:8954 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 736
  total_bytes: 11424
}

2021-10-02 11:55:03,304 INFO    MainThread:8954 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 11424
  total_bytes: 11424
}

2021-10-02 11:55:03,407 INFO    MainThread:8954 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 11424
  total_bytes: 11424
}

2021-10-02 11:55:03,510 INFO    MainThread:8954 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 11424
  total_bytes: 11424
}

2021-10-02 11:55:03,613 INFO    MainThread:8954 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 11424
  total_bytes: 11424
}

2021-10-02 11:55:03,769 INFO    MainThread:8954 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 11424
  total_bytes: 11424
}

2021-10-02 11:55:04,055 INFO    MainThread:8954 [wandb_run.py:_wait_for_finish():1697] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 11424
  total_bytes: 11424
}
local_info {
}

2021-10-02 11:55:05,421 INFO    MainThread:8954 [wandb_run.py:_show_files():1947] logging synced files
